ODIN_II/SRC/read_blif.cpp

```diff
@@ -535,7 +535,15 @@ void create_hard_block_nodes(hard_block_models *models, FILE *file, Hashtable *o
 	hard_block_ports *ports = get_hard_block_ports(mappings, count);
 
 	for (i = 0; i < count; i++)
+	{
+		if (mappings[i] != NULL)
```

Comment:
you just lost your pointer there.

set to NULL after free,
don't check if its NULL before setting to NULL, its a bit redundant

Reply:
https://www.eskimo.com/~scs/cclass/notes/sx11b.html